### ISSUE TITLE
version_converter failed

### ISSUE THREAD

**Issue by mengniwang95** [onnx/onnx#4060]
# Bug Report

### Describe the bug
This is the bug when I use version_converter to convert the model's opset from 10 to 12:
![image](https://user-images.githubusercontent.com/65003808/157160893-5cfc9817-7302-43d8-b564-bb9b8072ab14.png)
But actually this input is in model's initializer.
model: https://github.com/onnx/models/raw/main/vision/object_detection_segmentation/yolov3/model/yolov3-10.onnx

### System information
- OS Platform and Distribution (*e.g. Linux Ubuntu 16.04*): CentOS Linux release 7.5.1804 (Core)
- ONNX version (*e.g. 1.7*):  1.10 & 1.9
- Python version: 3.7.10

### Reproduction instructions
- Describe the code to reproduce the behavior.
```
import onnx
from onnx import version_converter
model = onnx.load('model.onnx')
new_model = version_converter.convert_version(model, 12)
```

**Comment by jcwchen** [comment]
Hi @mengniwang95,
I believe be it should be fixed by this PR: https://github.com/onnx/onnx/pull/3676, which is included in newer ONNX 1.11. Could you please try ONNX 1.11? Thanks.

**Comment by mengniwang95** [comment]
thank you very much, onnx1.11 can convert successfully
